frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverview.tsx

```diff
@@ -79,16 +84,30 @@ const MonitoringOverview: React.FC<MonitoringOverviewProps> = ({ resource, event
             Metrics
           </AccordionToggle>
           <AccordionContent id="metrics-content" isHidden={!expanded.includes('metrics')}>
-            <div className="odc-monitoring-overview__view-monitoring-dashboard">
-              <Link
-                to={`/dev-monitoring/ns/${resource?.metadata?.namespace}/?workloadName=${
-                  resource?.metadata?.name
-                }&workloadType=${resource?.kind?.toLowerCase()}`}
-              >
-                View monitoring dashboard
-              </Link>
-            </div>
-            <WorkloadGraphs resource={resource} />
+            {resource.kind === DeploymentConfigModel.kind ? (
+              <EmptyState>
+                <EmptyStateIcon
+                  icon={InfoCircleIcon}
+                  className="odc-monitoring-overview__empty-state-icon"
+                />
+                <EmptyStateBody className="odc-monitoring-overview__empty-state-body">
+                  Deployment Configuration metrics are not yet supported.
+                </EmptyStateBody>
+              </EmptyState>
```

Comment:
You are fighting against all the styles that make the `EmptyState` component useful: the icon size and color and spacing.

@openshift/team-devconsole-ux @serenamarie125 why is UX choosing to not use the PF empty state spacing and icon size?

Reply:
https://patternfly-react.surge.sh/documentation/react/components/emptystate

Reply:
cc @Veethika

Reply:
<img width="595" alt="Screenshot 2020-02-28 at 6 24 21 PM" src="https://user-images.githubusercontent.com/35127763/75550943-032b9b00-5a59-11ea-80eb-89aa5e882536.png">

 @christianvogt @sahil143 We could go with this design to adhere to PF standards. 
**Icon color** : -pf-globaliconColor-light
**font size**  title style  -  PF subtitle - pf-globalcolor-100 
body style  -  PF body - pf-globalcolor-200 
 cc @serenamarie125